dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -446,6 +446,12 @@ def generate_facilitator_averages(summary)
           next unless histogram_for_this_workshop.key? facilitator
         end
 
+        # The summary passed into this function (which was generated by
+        # generate_workshops_survey_summary) may contain num_respondent entries
+        # alongside the answer entries.  We should ignore such entries for the
+        # purposes of doing the calculations here.
+        histogram_for_this_workshop.except!("num_respondents")
```

Comment:
Do we have any test that could have caught this issue?

Reply:
Good question.  I updated one test in a new commit that fails without this fix, and succeeds with it.